src/components/AddActionDialog.vue

```diff
@@ -1,11 +1,15 @@
 <script setup lang="ts">
+import type { ActionElement } from '@/action'
 import actionStore from '@/actionStore'
+import { cloneDeep } from '@/helpers'
 import { ref } from 'vue'
+import { v4 as guid } from 'uuid'
 
 const props = defineProps(['action'])
```

Comment:
maybe we can type this to see if nullability is something we can use formally down in the file